tests/snapshots/index.js.md

```diff
@@ -25,6 +25,13 @@ Generated by [AVA](https://avajs.dev).
 > stdout
 
     `owner and repositories set, creating token for repositories "create-github-app-token" owned by "actions"␊
+    ␊
+    GET /api/v3/repos/actions/create-github-app-token/installation␊
+    ␊
+    ␊
+    POST /api/v3/app/installations/123456/access_tokens␊
+    "{\\"repositories\\":[\\"create-github-app-token\\"]}"␊
+    ␊
```

Comment:
We didn't include assertions for the requests our code was sending, this PR adds that to all our `main` fwtests